internal/runner/jest_test.go

```diff
@@ -299,6 +299,33 @@ func TestJestRetryCommandNameAndArgs_HappyPath(t *testing.T) {
 	}
 }
 
+func TestJestRetryCommandNameAndArgs_WithSpecialCharacters(t *testing.T) {
+	testCases := []string{"this will fail", "test with special characters .+*?()|[]{}^$"}
```

Comment:
How come this first string is called`"this will fail"`? It made me think that string was going to fail for some reason? But actually I think the intent of your test is to check that special characters are escaped correctly?

Reply:
I wanted to test if the `testNamePattern` is passed correctly if there is multiple test. I just used the same test name from other test. I'll change the test name to avoid confusion.